src/main/io/gps.c

```diff
@@ -227,7 +227,7 @@ gpsData_t gpsData;
 PG_REGISTER_WITH_RESET_TEMPLATE(gpsConfig_t, gpsConfig, PG_GPS_CONFIG, 0);
 
 PG_RESET_TEMPLATE(gpsConfig_t, gpsConfig,
-    .provider = GPS_NMEA,
+    .provider = GPS_UBLOX,
```

Comment:
This will break GPS support for all non-Ublox users - leaving the default at `GPS_NMEA` will still work for Ublox users and non-Ublox users, so this is what it should be.

Also, do you have data to back up your claim?

Reply:
Current NMEA implementation in BF does not support: SBAS, Dynamic platform model, Galileo, 3D Speed, sat list info (for the configurator).
I wonder how many people is using non-UBLOX modules for this to be a problem, they can change the setting after all.
It doesn't help either that the configurator shows the SBAS dropdown for NMEA, which doesn't work.

Reply:
Hiding the SBAS drop down in configurator sounds like a fix that is independent of this - changing this will not actually fix it.

Reply:
Indeed, but my point is that starting to use a gps module on BF today involves dealing with all this mess, this PR is just a first step.